chimba/chain.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "chimba",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://chimba.ooo/",
+  "pretty_name": "Chimba",
+  "chain_id": "chimba",
```

Comment:
Looks like the chain-id might be chimbanode-1, as per what I see on the explorer.

Reply:
chimbanode-1 is the name for moniker but the chain-id is chimba